pool/pool_server.py

```diff
@@ -233,14 +233,16 @@ async def get_login(self, request_obj) -> web.Response:
 
         self.pool.log.info(f"Login successful for launcher_id: {launcher_id.hex()}")
 
+        return await self.login_response(launcher_id)
```

Comment:
Can you call this method before it has been defined? Have you tested this?

Reply:
Yes and yes. The class and its methods are defined at import time. The method body is executed (thus all symbols are evaluated) at call time.

Reply:
Oh I didn't see the self there, I thought it was defined within the method. Thanks!